### PR TITLE
Update Axios

### PR DESCRIPTION
**Pull request by nagilson**
Please reach out for further questions.

### CODE REVIEW

File: sample/package.json [hunk 1]
```diff
@@ -112,6 +112,7 @@
 	},
 	"dependencies": {
 		"@vscode/vsce": "^2.19.0",
+		"axios": "^1.7.4",
```
Comment (nagilson) [inline sample/package.json:115]:
```suggestion
```

**Comment by nagilson** [comment]
Waiting for upstream azdo to be fixed, ICM issue

**Comment by nagilson** [comment]
ICM resolved

**Review by smitpatel — APPROVED** [review]

**Pull request merged**